octodns_selectel/__init__.py

```diff
@@ -12,7 +12,7 @@
 from octodns.provider import ProviderException
 from octodns.provider.base import BaseProvider
 
-__VERSION__ = '0.0.1'
+__VERSION__ = '0.0.2'
```

Comment:
I prefer to bump versions and cut releases in dedicated PRs. Once this has resolved/shipped I can run through that process. 

```suggestion
__VERSION__ = '0.0.1'
```

The CHANGELOG addition up above is fine, those often do happen along with the related changes.